frontend/pages/beta/index.tsx

```diff
@@ -0,0 +1,164 @@
+import SearchIcon from '@mui/icons-material/Search'
+import Box from '@mui/material/Box'
+import Chip from '@mui/material/Chip'
+import Grid from '@mui/material/Grid'
+import IconButton from '@mui/material/IconButton'
+import InputBase from '@mui/material/InputBase'
+import MuiLink from '@mui/material/Link'
+import Paper from '@mui/material/Paper'
+import Stack from '@mui/material/Stack'
+import { useTheme } from '@mui/material/styles'
+import Tab from '@mui/material/Tab'
+import Tabs from '@mui/material/Tabs'
+import Typography from '@mui/material/Typography'
+import Link from 'next/link'
+import React, { useState } from 'react'
+import TagSelector from 'src/common/TagSelector'
+
+import { ListModelType, useListModels } from '../../data/model'
+import EmptyBlob from '../../src/common/EmptyBlob'
+import MultipleErrorWrapper from '../../src/errors/MultipleErrorWrapper'
+import Wrapper from '../../src/Wrapper'
+import { Model, Version } from '../../types/types'
+import useDebounce from '../../utils/hooks/useDebounce'
+
+export default function ExploreModels() {
+  const [group, setGroup] = useState<ListModelType>('all')
+  const [filter, setFilter] = useState('')
+  const debouncedFilter = useDebounce(filter, 250)
+
+  const { models, isModelsError, mutateModels } = useListModels(group, debouncedFilter)
+
+  const theme = useTheme()
+
+  const error = MultipleErrorWrapper(`Unable to load marketplace page`, {
+    isModelsError,
+  })
+  if (error) return error
+
+  const handleFilterChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setFilter(e.target.value)
+  }
+
+  const onFilterSubmit = (e: React.FormEvent) => {
+    e.preventDefault()
+  }
+
+  const updateSelectedTags = (selectedTags: string[]) => {
+    console.log(selectedTags)
+  }
+
+  const updateSelectedType = (selectedType: string) => {
+    if (selectedType) {
+      switch (selectedType) {
+        case 'My models':
+          setGroup('user')
+          break
+        case 'Favourites':
+          setGroup('favourites')
+          break
+        default:
+          setGroup('all')
+      }
+      mutateModels()
+    }
+  }
+
+  return (
+    <Wrapper title='Explore Models' page='marketplace'>
+      <Paper
+        component='form'
+        onSubmit={onFilterSubmit}
+        sx={{
+          p: '2px 4px',
+          display: 'flex',
+          alignItems: 'center',
+          width: '70%',
```

Comment:
setting `width` like this seems a bit pointless when we're setting a `maxWidth` below

Reply:
This is actually left from the original marketplace, so I don't know why `width `and `maxWith `are set. I've actually gone and removed `width`.